### DISCUSSION TITLE
심화프로젝트 5주차 팀회고

### DISCUSSION THREAD

**Discussion by yeji0512** [codeit-bootcamp-frontend/2-Weekly-Mission#446] [회고]
**벌써 5주차가 마무리 되어가네요:)
저번주 회고에서는 각 팀별로 제가 코멘트를 드렸는데요, 
회고를 읽어보니 모두 이 프로젝트에 얼마나 진심인지 느낄 수 있었어요. 정말 멋있습니다 🔥** 

**마지막 회고네요. 
막판 스퍼트! 남은 한주의 프로젝트 기간도 즐겁게 달려보아요 🏃‍♀️**

# 왜 회고를 해야하나요?
- **원활한 팀 프로젝트 진행을 위해, 회고가 좋은 도구가 되어줄거예요.** 
매주 팀 단위로 회고를 하며 어떤 점이 좋았고, 어떤 점이 아쉬웠고, 아쉬웠던 부분은 어떻게 개선해볼지 함께 이야기해보면 
그 다음주에는 더 나은 프로젝트를 진행할 수 있을거예요. 회고를 통해 함께 성장해보아요 🔥 
- **회고 시간만큼은 솔직한 생각, 마음을 터놓아보기! 😎** 
솔직한 회고를 통해 다음주는 더 즐겁게 프로젝트를 진행할 수 있을거예요.  

# 회고 방법
![image](https://github.com/codeit-bootcamp-frontend/2-Weekly-Mission/assets/144666341/b1f1cc83-7d24-4156-910d-3fc9b6bb7555)
회고는 **“KPT 회고”** 방식으로 진행해볼거예요. 아래 방법대로 회고해주시면 됩니다. 
- 팀원들과 모여 그 주의 팀 프로젝트 활동을 돌아보며, '좋았던 점(Keep) / 아쉬웠던 점(Problem) / 시도해볼 것(Try)'으로 항목을 나누어 회고해주세요.
    - Keep: 현재 우리팀이 잘 하고 있어서 계속 유지하면 좋을 것 같은 부분
    - Problem: 현재 우리팀이 가지고 있는 문제
    - Try: Problem에 적은 것을 해결하기 위해 우리 팀이 다음에 시도해볼만한 것 (혹은 Keep 중에서도 더 개선하거나 시도해보고 싶은 것)
- [KPT 회고 피그마 템플릿](https://www.figma.com/file/0NHIhlM9vhYZDrtZa7nJn8/%ED%9A%8C%EA%B3%A0-%ED%85%9C%ED%94%8C%EB%A6%BF?type=whiteboard&node-id=0%3A1&t=p6vXDVEjQntHKaNP-1)을 활용해서, 팀 회고를 진행하셔도 됩니다. 단 템플릿 사용시, 팀별로 복제해서 사용해주세요. 꼭 템플릿을 사용하지 않아도 되니, 팀별로 편한 방식으로 회고를 진행해주세요. 
- **이번주 일요일까지, 팀 회고한 내용을 정리하여 댓글로 올려주세요.** 
(댓글에 예시 회고를 올려드릴테니, 참고해주세요. 피그마 템플릿으로 진행하셨다면 캡처본을 올려주셔도 돼요.)

# 참고 글
- [KPT 회고란 무엇인가?](https://brunch.co.kr/@jinha0802/35)
- [팀문화의 탄생](https://techblog.woowahan.com/2677/)

**Comment by CheeseB** [comment]
# 3팀 🦸🏼 

![image](https://github.com/codeit-bootcamp-frontend/2-Weekly-Mission/assets/43297823/bab1e2cd-040c-4a97-933d-4ff8f53b0a6f)

**Comment by muyahho** [comment]
- 5주차 회고
    
### 4팀
    
### Keep
    
    먼저 작업이 끝난 팀원들이 적극적으로 추가 작업을 하거나 다른 팀원을 도와주며 협력을 잘 해나가고 있다.
    매일 회의록을 작성합니다.
    소통이 원활히 진행됩니다.
    문제 해결을 위한 협력이 활발합니다.
    변함없이 열심히 하려고 합니다.
    요구사항을 다 구현하고 빨리 잘 마무리했습니다. 
    
### Problem
    
    컨벤션들을 제대로 지키지 않은 부분이 있는것 같습니다.
    코드리뷰를 시간부족으로 꼼꼼히 못했습니다. 
    PR올릴 때 코드량을 한꺼번에 많이 올려서 전체를 파악하기 힘들었습니다. 
    스웨거, 요구사항을 정확히 확인 안하고 작업을 해서 재작업하는 경우가 있었습니다. 
    
 ### Try
    
    컨벤션 제대로 지키기
    적극적으로 코드 리뷰 하기
    요구사항을 정확히 확인해서 리팩토링 하기
    PR을 자주 올리기
    
    마지막까지 화이팅!!

**Comment by mijin0928** [comment]
## 5팀
### Keep
- 꼼꼼하게 작업을 잘 진행하고 있는 것 같습니다
- 작업이 늦어지는 팀원들을 잘 도와줍니다

### Problem
- 생각보다 완성이 늦어지는 것 같습니다

### Try
- pr 확인 빨리하고 pr리뷰 힘들더라도 꼼꼼히 확인해주기
- 시간 상관없이 의견 공유하고 싶은 부분이 있으면 바로 공유하기

![2539a702f2eba1b1](https://github.com/codeit-bootcamp-frontend/2-Weekly-Mission/assets/94034865/01b5a473-5595-40e2-953e-39cb1cf709bc)

**Comment by 9ummy9** [comment]
# 6팀

![스크린샷 2024-03-31 114234](https://github.com/codeit-bootcamp-frontend/2-Weekly-Mission/assets/148167964/53ceccc7-1a6a-4e63-acff-f41f29cdbb73)



----

![JitteryHorribleBoto-size_restricted](https://github.com/codeit-bootcamp-frontend/2-Weekly-Mission/assets/148167964/98b247ef-bc3d-4980-9b2a-cdeec4ab303e)


발표일까지 파이팅!!!!

**Comment by JISOuO** [comment]
# 10팀😎
## Keep
- 트러블 슈팅을 꼼꼼하게 작성중이다
- PR컨벤션을 잘 지켜 올린다
- 문제가 생기면 다같이 해결한다
- 여러방면으로 문제를 고민해본다 

## Problem
- 생각이 많아서 회의 시간이 길어진다
- 프로젝트 기간이 추가 될 수도 있을 것 같다

## Try
- 트러블 슈팅 쌓인 이슈들 해결해보기
- 프로젝트, 발표 마무리 잘 하기 

마지막 파트는 10팀이 점령하겠습니다+_+

**Comment by rladngus133** [comment]
## 2팀
### Keep
- 5주동안 팀미팅 시간이 잘 지켜지고 불참이 없다.
- 각 팀원들이 코드를 자신것만이 아닌 전체적으로 고려해주는게 좋았다.
- 프로젝트 마무리시기까지 코드리뷰없이 PR되는 경우가 없어 좋았다.
- PR , 코드등 컨벤션이 잘 지켜져서 좋았다.
- 문제 , 트러블 발생시 다 같이 고민하는 팀 분위기가 좋았다.

### Problem
- 기능개발이후 코드리뷰하기가 어려워졌다...

### Try
- 개인 시간분배를 잘해야 할 것 같다.
